### DISCUSSION TITLE
Welcome to react-swipeable-list Discussions!

### DISCUSSION THREAD

**Discussion by marekrozmus** [marekrozmus/react-swipeable-list#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SukhorukovS** [comment]
Hello. How can I programmaticaly hide actions? For example when swiping another element, or scrolling the page?

**Reply by marekrozmus** [reply]
Hi, could you please describe the problem a little bit more? What icons?

**Reply by Starly26** [reply]
Hello, I have the same problem. For example, when I long press on an element, I want to pro grammatically close the shown actions.I think we need to add a callback calling playReturnAnimation
